tests/__init__.py

```diff
@@ -173,6 +173,9 @@ def has_pyftpdlib():
 @lru_cache(1)
 def has_biplist():
     """Test if biplist is available."""
+    if sys.version_info >= (3, 9):
+        return False
+
```

Comment:
Why not import `has_biplist` from `linkcheck.bookmarks.safari` instead of duplicating the logic?

Reply:
Reused the linkcheck.bookmarks.safari function.